.github/workflows/ci.yml

```diff
@@ -10,7 +15,7 @@ jobs:
     - name: config
       run: ./config --strict-warnings
```

Comment:
Could we add in each `config` run line also ` && perl configdata.pm --dump` so we can inspect most of the environment to debug the weird failures?

Reply:
I've done this now.
Would be nice if this could be done globally, but as a CI scrip novice I don't know how to achieve that.